navigation-api/navigate-event/navigation-traverseTo-same-document-preventDefault-multiple-windows.html

```diff
@@ -19,7 +20,7 @@
 
   navigation.onnavigate = e => e.preventDefault();
   i.contentWindow.navigation.onnavigate = t.unreached_func("navigate event should not fire in the iframe, because the traversal was cancelled in the top window");
-  await promise_rejects_dom(t, "AbortError", navigation.traverseTo(navigation.entries()[start_index].key).finished);
+  assertBothRejectDOM(t, navigation.traverseTo(navigation.entries()[start_index].key), "AbortError");
```

Comment:
You have to await this, no? I think it ends up being largely ignored otherwise.

Reply:
Oops, well caught, indeed it needs an await.